libpod/runtime.go

```diff
@@ -151,6 +151,7 @@ var (
 			"/usr/lib/cri-o-runc/sbin/runc",
 		},
 		ConmonPath: []string{
+			"/usr/libexec/podman/conmon",
```

Comment:
This seems unrelated to rootless-ness?

Reply:
it is related since at the moment I don't read the system configuration file (as they use paths that are not writeable to a non root user).  The default configuration file already has this path, probably it was just not updated here.

Reply:
> The default configuration file already has this path, probably it was just not updated here.

Sounds like this commit could be pulled out into a separate PR and landed immediately then.  Is there a reason to hold it back with the rootless stuff?

Reply:
moved to: https://github.com/projectatomic/libpod/pull/872